<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartMenu</class>
 <widget class="QWidget" name="StartMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Bg_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../assets/assets.qrc">:/Items/titlescreen_new.jpg</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ChooserBg">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>175</y>
     <width>360</width>
     <height>325</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../assets/assets.qrc">:/Items/SeedChooser_Background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>205</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="StartBotton">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="SettingButton">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ExitButton">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>270</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Papyrus</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border: none;
    border-image: url(:/Items/ui/SeedChooser_Button.png) 0 0 0 0 stretch stretch;
}

QPushButton:hover {
    border-image: url(:/Items/ui/SeedChooser_Button2.png) 0 0 0 0 stretch stretch;
}

QPushButton:pressed {
    border-image: url(:/Items/ui/SeedChooser_Button_Disabled.png) 0 0 0 0 stretch stretch;
}</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
